docs/4.x/archiving.md

```diff
@@ -41,21 +41,24 @@ While pre-computing archive data seems of course superfluous for 1000 page views
 
 ## When?
 
-By default, archive data is calculated and cached **on-demand**. When a specific report is requested, Piwik will check if the required archive data exist and generate it if not.
+### On-demand Archiving
+
+By default, archive data is calculated and cached **on-demand**. When a specific report is requested, Piwik will check if the required archive data exists and generate it if it does not.
 
 ### Pre-archiving
 
-When tracking a website with a lot of traffic, the archiving on-demand might take too much time. In those situations, archiving on demand must be disabled and [pre-archiving needs to run in background at a scheduled time](https://matomo.org/docs/setup-auto-archiving/).
+When tracking a website with a lot of traffic, on-demand archiving will take too much time and resources, causing users to wait a long time before a report becomes visible.
```

Comment:
Maybe we could be more specific and say something like:

if you have more than 500 page views a day, or a slow server, or generally find Matomo slow, then we recommend you disable browser archiving and set up CLI archiving...

Reply:
👍